ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -324,6 +328,21 @@ private MainnetProtocolSpecs() {}
         .name("MuirGlacier");
   }
 
+  // TODO EIP-1559 change for the actual fork name when known
+  static ProtocolSpecBuilder<Void> eip1559Definition(
+      final Optional<BigInteger> chainId,
+      final OptionalInt contractSizeLimit,
+      final OptionalInt configStackSizeLimit,
+      final boolean enableRevertReason,
+      final GenesisConfigOptions genesisConfigOptions) {
```

Comment:
Maybe start this off with a check of the feature flag?  Throw an exception if not set?

Reply:
Good idea.

Reply:
Done